services/src/main/java/io/druid/cli/CliOverlord.java

```diff
@@ -118,7 +118,7 @@
 {
   private static Logger log = new Logger(CliOverlord.class);
 
-  private static List<String> UNSECURED_PATHS = Lists.newArrayList(
+  protected static List<String> UNSECURED_PATHS = Lists.newArrayList(
```

Comment:
since this is going `protected` can it be an `ImmutableList.of`?

Reply:
sure, changed this to ImmutableList